settings.gradle

```diff
@@ -5,6 +5,170 @@ dependencyResolutionManagement {
         mavenCentral()
         maven { url 'https://jitpack.io' }
     }
+
+    versionCatalogs {
+        libs {
+            // Judging the impact of newer library versions on the app requires being intimately familiar with the code
+            // base. Please don't open pull requests upgrading dependencies if you're a new contributor.
+            version('androidGradlePlugin', '7.3.1')
+            version('kspGradlePlugin', '1.7.22-1.0.8')
+            version('ktlintGradlePlugin', '11.0.0')
+
+            version('kotlin', '1.7.22')
```

Comment:
Please only use a version definition if two or more artifacts share the same version. For most of the dependencies you can include the version in the library definition below.

Reply:
done